cppcache/include/geode/ExceptionTypes.hpp

```diff
@@ -812,6 +812,18 @@ class APACHE_GEODE_EXPORT PutAllPartialResultException : public Exception {
   }
 };
 
+/**
+ *@brief Thrown when an error is encountered during an SSL operation.
+ **/
+class APACHE_GEODE_EXPORT SSLException : public Exception {
+ public:
+  using Exception::Exception;
+  ~SSLException() noexcept override {}
```

Comment:
`SslException` would be preferable for camel casing.

Reply:
except that `SSL` is an acronym... didn't think we liked to camel them...

Reply:
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules
> For some symbols, this style guide recommends names to start with a capital letter and to have a capital letter for each new word (a.k.a. "Camel Case" or "Pascal case"). When abbreviations or acronyms appear in such names, prefer to capitalize the abbreviations or acronyms as single words (i.e `StartRpc()`, not `StartRPC()`).

Reply:
fair nuf!

Reply:
Done